src/Section2/style2.js

```diff
@@ -15,16 +24,28 @@ export const SubTitle = styled.h2`
   font-size: 50px;
   color: #3f3d56;
   padding-bottom: 35px;
+
+  @media (max-width: 768px) {
+    font-size: 45px;
+    color: #3f3d56;
+    padding-bottom: 32px;
+  }
+
+  @media (max-width: 376px) {
+    font-size: 30px;
+    color: #3f3d56;
+    padding-bottom: 35px;
+  }
 `;
 
 export const Block0 = styled.div`
-display: flex;
-flex-direction: row;
-flex-align: center;
-justify-content: center;
-flex-wrap: wrap;
-gap:28px
-`
+  display: flex;
+  flex-direction: row;
+  flex-align: center;
+  justify-content: center;
+  flex-wrap: wrap;
+  gap: 28px;
+`;
 
 export const Block1 = styled.div`
```

Comment:
You could be more descriptive with the names you give to your components, this would help come back and remember the context they are used in.
This could be something like: `SkillBox`